docs/src/understand/sft.rst

```diff
@@ -26,41 +30,41 @@ without compromising end-to-end security.
 Architecture
 ------------
 
-The following diagram is centered around SFT and its role within a calling setup. Restund is seen
+The following diagram is centered around `SFT` and its role within a calling setup. Restund is seen
```

Comment:
We won't be using `restund` much longer. `coturn` it'll be (Joe is working on deploying it to production.)